threadgroup.h

```diff
@@ -0,0 +1,58 @@
+#ifndef CUSTOM_THREAD_H
+#define CUSTOM_THREAD_H
```

Comment:
Make include guards consistent with file name

Reply:
Good point -- in fact all include guards should now have 'DYSCO' in front of them, to make sure they won't clash with casacore. I've changed this one to DYSCO_THREADGROUP_H and also added DYSCO_ in front of those that didn't have it.